game/run_web.sh

```diff
@@ -4,4 +4,6 @@ set -e
 wasm-pack build --dev --target web -- --no-default-features --features wasm
 cp index.html pkg
 cd pkg
+rm -f system
+ln -s ../../data/system/ .
```

Comment:
alternatively you could have the webroot checked in with a symlink. I'm a little nervous about a script which deletes files - especially when they don't enforce what directory you're currently in.

Reply:
Good call, done. `wasm-pack` sticks a `.gitignore` in `pkg/`, but turns out you can delete it, add the symlinks, then let the gitignore file come back, and it all seems to work.